src/databricks/labs/lsql/dashboards.py

```diff
@@ -728,6 +766,118 @@ def _get_query_widget_spec(fields: list[Field], *, frame: WidgetFrameSpec | None
         return spec
 
 
+@dataclass
+class FilterTile(Tile):
+    _position: Position = dataclasses.field(default_factory=lambda: Position(0, 0, 3, 2))
+
+    def validate(self) -> None:
+        """Validate the tile
+
+        Raises:
+            ValueError : If the tile is invalid.
+        """
+        if not self.metadata.is_filter():
+            raise ValueError(f"Tile is not a filter file: {self}")
+        if len(self.metadata.filters) == 0:
+            raise ValueError(f"Filter tile with path {self.metadata.path} has no filters defined")
```

Comment:
Keep consistent error messages

```suggestion
            raise ValueError(f"Filter tile has no filters defined: {self}")
```